From 7391bb6ff5b4c18a5a194732eeea6e960becb1fc Mon Sep 17 00:00:00 2001
From: Paul Naoumenko <paulx.naoumenko@intel.com>
Date: Mon, 14 Oct 2013 17:00:05 +0200
Subject: [PORT FROM MAIN] wifi: Disable TDLS in android.config

BZ: 130080

TDLS could be dependent to the WiFi driver/FW and does not appear in the CDD
For safety we disable it in our supplicant confifuration file

TODO:
Need to be re-activated once we got confirmation from our vendors that TDLS
is supported at driver/FW level.

Change-Id: Ia678bb904e9e5d187c4e358f322fed8669a57949
Orig-Change-Id: Ide80f90db5c3b2669616033633e428a7b4a076b0
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Jean-Michel Bachot <jean-michel.bachot@intel.com>
---
 wpa_supplicant/android.config |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wpa_supplicant/android.config b/wpa_supplicant/android.config
index 93fe809..d2e737d 100644
--- a/wpa_supplicant/android.config
+++ b/wpa_supplicant/android.config
@@ -510,7 +510,7 @@ CONFIG_AP=y
 # more information on P2P operations.
 CONFIG_P2P=y
 
-CONFIG_TDLS=y
+#CONFIG_TDLS=y
 
 #Enable Wifi Display
 CONFIG_WIFI_DISPLAY=y
-- 
1.7.4.1

